validation/scripts/validate-plot-protein-lengths.R

```diff
@@ -46,40 +40,34 @@ diamond_proportion_plot <- diamond_tables_clean %>%
 
 diamond_proportion_plot
 
+# number of proteins with greater than 90% length proportion of reference hit
 diamond_tables_clean %>%
   left_join(tick_species_metadata) %>% 
   mutate(proportion = slen / qlen) %>% 
-  filter(proportion >=0.9) %>% 
+  filter(proportion > 0.9) %>% 
   group_by(species_name) %>% 
   count() %>% 
   left_join(tick_species_metadata) %>% 
   mutate(proportion_of_reference = n / total_protein_count) %>% 
   arrange(desc(proportion_of_reference))
 
-diamond_tables_clean %>%
-  left_join(tick_species_metadata) %>% 
-  mutate(proportion = slen / qlen) %>% 
-  filter(proportion <0.9) %>% 
-  group_by(species_name) %>% 
+# amblyomma proteins with more than 2 hits
+diamond_tables_clean %>% 
+  group_by(species_name, sseqid) %>%
   count() %>% 
-  left_join(tick_species_metadata) %>% 
-  mutate(proportion_of_reference = n / total_protein_count) %>% 
-  arrange(desc(proportion_of_reference))
-
-# combine with tick metadata for total protein counts
-tick_species_protein_hit_counts <- diamond_tables_clean %>% 
+  filter(n > 2) %>% 
   group_by(species_name) %>% 
   count() %>% 
-  mutate(protein_hits = n) %>% 
-  select(-n)
+  mutate(proportion_of_amblyomma = (n / 28319) * 100) # divided by total proteins in A. americanum for more than 2 hits
+
```

Comment:
this diff is hard to follow -- you don't need to filter to below .9 this time since ...? or is it doing something different than before? (also it looks like this isn't saved just printed out so probs fine no matter what

Reply:
Interesting, I think before I wanted some stats and was playing around with what number of proteins had length proportions greater than 90% of the reference protein, and I would change that number to see different distributions. I must have reuused that part to get this more than 2 protein hits within a reference. But yes was just printing to see different subsets of results

Reply:
I had quite a few of these non-saved lines of code and just printing to see how things looked and I cleaned that up, so I think that's what happened here but I wanted to keep the filtering for more than 2 hits since that final check became important